target/linux/generic/files/drivers/mtd/mtdsplit/mtdsplit_uimage.c

```diff
@@ -31,11 +31,20 @@
 #define IH_MAGIC	0x27051956	/* Image Magic Number		*/
 #define IH_NMLEN		32	/* Image Name Length		*/
 
-#define IH_OS_LINUX		5	/* Linux	*/
+#define IH_OS_LINUX		5	/* Linux			*/
```

Comment:
Any reason for that?

Reply:
The reason is consistency with lines on the top and below

Reply:
Since this is not adjacent to any code you change, I'd just have kept it as it is.

But if you prefer it changed, I won't stop you.

Reply:
it was the only comment which did not follow the pattern on the top and the bottom so I decided to change it